cyipopt/tests/unit/test_scipy_ipopt_from_scipy.py

```diff
@@ -239,127 +222,8 @@ def test_minimize_bound_equality_given2(self):
                        options=self.opts)
         assert_(res['success'], res['message'])
         assert_allclose(res.x, [0.8, 0.8], atol=1e-3)
-        assert_(-0.8 <= res.x[0] <= 1)
-        assert_(-1 <= res.x[1] <= 0.8)
-
-    # fmin_slsqp
```

Comment:
Delete all these `fmin_slsqp` tests; we only want tests of the `minimize` interface.